contracts/NodeDelegatorLST.sol

```diff
@@ -389,8 +396,8 @@ contract NodeDelegatorLST is
         // Calculate the amount of assets returned from the withdrawal
         assets = IERC20(asset).balanceOf(address(this)) - assetsBefore;
         if (assets > 0) {
-            // transfer withdrawn assets to the staker
-            IERC20(asset).transfer(staker, assets);
+            // transfer withdrawn assets to the receiver, which is either the staker or the LRTDepositPool contract
+            IERC20(asset).transfer(receiver, assets);
         }
 
         emit ClaimWithdrawal(address(withdrawal.strategies[0]), withdrawalRoot, staker, assets);
```

Comment:
nit: Should this event be updated to include `receiver` param as well?

Reply:
`ClaimWithdrawal` is an existing event hence I was reluctant to change it and break indexers and analytics systems. I think Dune can handle event changes like this, but I'm not sure about Squid which is used for the frontend. I'll ask the data team

Reply:
`ClaimWithdrawal` is also used by `claimInternalWithdrawal`. The `receiver` in that case would be the DepositPool contract.

Reply:
The Frontend isn't using the `ClaimWithdrawal` event so I've added the `receiver` param. It's not indexed as we already have three fields indexed which is the max.